### PR TITLE
chore: release ci cleanup

### PR DESCRIPTION
**Pull request by sash-a**
Reconfigured the workflow, requirements, tests and the like. Mostly copying jumanji were appropriate

### CODE REVIEW

**Comment by sash-a** [comment]
Probably going to need merge this in after all the old mava stuff has been deleted, otherwise it won't pass the checks

File: .gitignore [hunk 1]
```diff
@@ -135,33 +135,31 @@ dmypy.json
...
 logs
 Checkpoints
-mava/workspace.code-workspace
 .DS_Store
-mava_testing
 .devcontainer/devcontainer.json
+3.9
+
+# smac
 3rdparty
```
Comment (RuanJohn) [inline .gitignore:155]: Since we won't be supporting smac we can remove this for now.

File: setup.py [hunk 2]
```diff
@@ -13,129 +12,55 @@
...
-        "record_episode": record_episode_requirements,
-        "sc2": smac_requirements,
-        "envs": pettingzoo_requirements + smac_requirements,
-        "jax": jax_requirements,
+        "dev": _parse_requirements("requirements/requirements-dev.txt"),
     },
+    package_data={"mava": ["py.typed"]},
     classifiers=[
-        "Development Status :: 3 - Alpha",
+        "Development Status :: 4 - Beta",
```
Comment (RuanJohn) [inline setup.py:54]: No exactly sure about this? Aren't we in alpha at the moment?

**Review by RuanJohn — CHANGES_REQUESTED** [review]
Thanks @sash-a! What a mammoth job! This looks super clean!
Just a few minor requests from my side.

**Review by RuanJohn — APPROVED** [review]
Looks good to me! Thanks @sash-a :fire:

**Comment by RuanJohn** [comment]
> 

Very good points, just some comments on them: 
* The troubleshooting will get removed in the readme PR. 
* For now, let's hold off on doing this and have one PR once all the systems are in where we move things around. 
* Very good catch. Can we please bring the Mava logo back @sash-a? My suggestion is that we keep it in the docs folder in `docs/images/`.

**Review by RuanJohn — APPROVED** [review]
Thanks @sash-a :raised_hands:

**Pull request merged**